src/labeler.ts

```diff
@@ -243,32 +257,22 @@ function checkMatch(
   return true;
 }
 
-async function addLabels(
+function removeLabel(labels: string[], label: string): void {
```

Comment:
The current name confuses me a little. Based on its name, I think it still makes an API call. Can we make this more specific?
```suggestion
function removeLabelFromList(labels: string[], label: string): void {
```

Reply:
Makes sense, done!